package.json

```diff
@@ -57,7 +57,7 @@
       }
     },
     "start-dev": {
-      "command": "npm run clean && concurrently --kill-others 'npm run webpack-dev-server' 'node bin/server.js'",
+      "command": "npm run clean && concurrently --kill-others 'npm run webpack-dev-server' 'node bin/server.js | ./node_modules/.bin/bunyan'",
```

Comment:
npm adds `./node_modules/bin` to the PATH automatically so I think this can just be `bunyan`